geer26/Simple_rest_api/routes/get_books.go

```diff
@@ -0,0 +1,135 @@
+package routes
+
+import (
+	"encoding/json"
+	"main/database"
+	"net/http"
+
+	"github.com/google/uuid"
+)
+
+func HandleBooks(w http.ResponseWriter, r *http.Request) {
+
+	switch r.Method {
```

Comment:
```suggestion
	switch r.Method {
	case "GET":
		getBook(w, r)
	case "PUT":
		updateBook(w, r)
	case "DELETE":
		deleteBook(w, r)
	case "POST":
		addBook(w, r)
	default:
		http.Error(w, "invalid method", http.StatusBadRequest)
	}
```

I think this would do the same work